src/viewport/api/public.py

```
@@ -101,6 +102,8 @@ async def _build_public_gallery_response(
     parent_share_id: UUID | None = None,
     record_view: bool = True,
     project_navigation: PublicProjectResponse | None = None,
+    override_appearance: PublicGalleryAppearance | None = None,
+    override_cover: Any = None,
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `override_cover`

(ANN401)